junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/ParallelExecutionConfiguration.java

```diff
@@ -60,4 +61,16 @@
 	 */
 	int getKeepAliveSeconds();
 
+	/**
+	 * Get the predicate called when there the ForkJoinPool is saturated.
+	 * This occurs when the ForkJoinPool has reached MaxPoolSize
+	 * and there are no threads available. The predicate
+	 * should return true if the blocking thread should try again.
+	 * If the predicate is null or returns false a RejectedExcecutionException
+	 * is thrown.
+	 * @return a Predicate accepting a ForkJoinPool as the parameter or null.
+	 * @since 5.9
+	 */
+	Predicate<? super ForkJoinPool> getSaturatePredicate();
```

Comment:
Please add an `@API` indication here as well

Reply:
Like this: `@API(since = "1.9")`, or 5.9? In addition to the `@since` or instead of it?

Reply:
In addition to the `@since` ;)

And you're right, the version should be `1.9` because this is in the platform module.